src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -332,12 +346,15 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
       }
       case 'Send': {
         if (!inboundAddressData || !accountAssetAddress) return undefined
-        const memo = `+:${thorchainNotationAssetId}:${otherAssetAddress ?? ''}:ss:${
-          confirmedQuote.feeBps
-        }`
-
+        const amountCryptoPrecisionWithMaybeDust = isDeposit
```

Comment:
Would prefer `or` terminology as this is implying we *add* dust to the value
```suggestion
        const amountOrDustAmountCryptoPrecisionDust = isDeposit
```

Reply:
https://github.com/shapeshift/web/pull/6285/commits/32f59bac73d2f81b3cd1f84ddbbf2c6a3244aa8a